src/components/Dialogs/utils.tsx

```diff
@@ -66,10 +66,10 @@ interface ExploreMetricModalMapContent {
 export const exploreMetricToFooterContentMap: {
   [key: number]: ExploreMetricModalMapContent;
 } = {
-  [ExploreMetric.DEATHS]: {
-    metricName: 'daily deaths',
-    howItsCalculated: 'Our daily deaths number is a seven-day average.',
-    metricDefinition: 'This is the number of COVID deaths per day.',
+  [ExploreMetric.WEEKLY_DEATHS_PER_100K]: {
+    metricName: 'weekly deaths',
+    howItsCalculated: 'Our weekly deaths number is a seven-day average.',
```

Comment:
I think this should be something like, "Our weekly deaths number is a rolling seven-day sum."

Reply:
This would be different from the content in the [copy doc](https://docs.google.com/document/d/1pITjM1gctulpwyqez5SFrQLFIYO4vndJArDjFPnVMG4/edit#heading=h.99jyctq40phz), under "Methodology tooltips" on p. 30. But I agree that yours is more accurate. I'll post a comment in the copy doc.

Reply:
Perfect, thanks!

Reply:
Anna just responded. We'll go with your suggestion.